### DISCUSSION TITLE
How to fix this garbled and How to edit this displayed text

### DISCUSSION THREAD

**Discussion by willyliu0202** [jagrosh/MusicBot#927] [Questions] [answered]
![IMG_20210928_113132.jpg](https://user-images.githubusercontent.com/56711425/135018944-9fb42c4f-a92a-49bc-b711-6ae127c0c71f.jpg)

**Comment by jagrosh** [comment, marked as answer]
Use unicode encoding for your config file, and use valid emojis in the config for the success/warning/error emojis

**Reply by willyliu0202** [reply]
thank you,I have solved it
